code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -1,5 +1,13 @@
 #define ELZUOSE_EMAG_COLORS list("#00ffff", "#ffc0cb", "#9400D3", "#4B0082", "#0000FF", "#00FF00", "#FFFF00", "#FF7F00", "#FF0000")
 
+#define GOOD_SOIL list(/turf/open/floor/plating/grass, /turf/open/floor/plating/dirt, /turf/open/floor/ship/dirt, /turf/open/floor/grass/ship, /turf/open/floor/plating/asteroid/whitesands/grass, /turf/open/floor/grass/fairy/beach)
```

Comment:
```suggestion
#define GOOD_SOIL list(/turf/open/floor/plating/grass, /turf/open/floor/plating/dirt, /turf/open/floor/plating/asteroid/whitesands/grass, /turf/open/floor/grass/fairy/beach)
```
This is more like astroturf than grass, just make sure it's all of the planetary ""soil"" types.

Reply:
Those are the grass/dirt subtypes the SRM ships use AFAIK which is lorewise proper soil? It's turf/open/grass that's the astroturf